src/components/customer-resource-show.js

```diff
@@ -136,6 +136,7 @@ export default class CustomerResourceShow extends Component {
             <PaneMenu>
               <IconButton
                 icon="edit"
+                title={`Edit ${model.name}`}
```

Comment:
So I know we have no control over this right now but `title` is not for a11y. Here's a great resource as to why: https://a11yproject.com/posts/title-attributes/

But the TL;DR screen readers treat `title` differently. If you want to label something use `aria-label`

Reply:
Lucky for us... `IconButton` has an `ariaLabel` prop. Changed.

Reply:
Oh good! I'd like to deprecate the `title` prop entirely. It should pretty much never be used.